samza-core/src/main/scala/org/apache/samza/config/JobConfig.scala

```diff
@@ -81,6 +81,9 @@ object JobConfig {
   val PROCESSOR_ID = "processor.id"
   val PROCESSOR_LIST = "processor.list"
 
+  // job level configuration for base directory for state stores
```

Comment:
Comment unnecessary, config name should be self-explanatory.

Reply:
fixed